deployment/utils.py

```diff
@@ -43,6 +43,11 @@ def load_tags():
     if 'owner' not in tags:
         tags['owner'] = os.popen("git config --get user.email").read().rstrip()
 
+    for k, v in tags.items():
+        if v is None:
+            raise RuntimeError(
```

Comment:
Should we simply ignore the tags with no value instead?

Reply:
yeah i quite like that actually. Should be just as straightforward to do.

Reply:
As we discussed last week, raising an error for tags with no value is actually more useful. I'm ready to merge when you fix the formatting nitpick below.